### ISSUE TITLE
Missing return type in generated $meta-objects

### ISSUE THREAD

**Issue by ruegerj** [vlvtin/vlquery#13]
The `$meta` objects, generated by the context-generator are lacking a return type on the getter `get set() {}`. This results in following error when the tsc compile option _"noImplicitAny"_ is set to `true`: 
_"error TS7022: 'set' implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer."_

This could be resolved by changing the following [line of the context-generator](https://github.com/levvij/vlquery/blob/master/src/tools/context-generator.ts#L230) to:
```ts
set: DbSet<${convertToClassName(table)}, ${convertToQueryProxyName(table)}> {
``` 
or by mentioning in the docs that the _"noImplicitAny"_ option must be disabled in order to use the generated context.

**Comment by levvij** [comment]
Fixed in v2.9.1!

**Comment by ruegerj** [comment]
Return type now is generated for the mentioned getter. All tough there seems to be a new little issue: now following line will be generated for a `$meta` object:
```ts
get set(): DbSet<Entity, EntityQueryProxy> { {
```
The issue is, that the double `{` leads to a compilation errors for each generated entity representation.

**Comment by levvij** [comment]
Fixed in v2.9.2!

**Comment by ruegerj** [comment]
Issue resolved, works now like a charm!
